langchain/agents/agent_toolkits/powerbi/chat_base.py

```diff
@@ -43,11 +45,12 @@ def create_pbi_chat_agent(
         llm=llm,
         tools=tools,
         system_message=prefix.format(top_k=top_k),
-        user_message=suffix,
```

Comment:
It looks like this may have been a bug?

Reply:
@hwchase17 please confirm if you have a chance

Reply:
yup think youre right!